src/svg.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.svg" {
+    const content: any;
```

Comment:
It should probably be this:
```
  const content: React.FunctionComponent<React.SVGAttributes<SVGElement>>;
```
I took it from here:
https://stackoverflow.com/questions/44717164/unable-to-import-svg-files-in-typescript

Didn't know that we need to do this. Thank you!